awsecs/awsecs.go

```diff
@@ -124,7 +124,7 @@ type updateTaskDefResult struct {
 
 // updateTaskDef creates a new task def revision with the container image updated to use the new Git SHA.
 // It returns the new ARN and previous Git SHA.
-func updateTaskDef(taskDefARN, gitsha, updateStrategy string, ecsClient ecsiface.ECSAPI) (updateTaskDefResult, error) {
+func updateTaskDef(taskDefARN, gitsha string, containers []string, updateStrategy string, ecsClient ecsiface.ECSAPI) (updateTaskDefResult, error) {
```

Comment:
Nit, why not add `containers` after `updateStrategy`? Then you can keep the first 3 params all having the same type.